hudi-aws/src/test/java/org/apache/hudi/aws/testutils/GlueTestUtil.java

```diff
@@ -112,7 +112,7 @@ public static void createHoodieTable() throws IOException {
 
     String instantTime = "101";
     HoodieCommitMetadata commitMetadata = new HoodieCommitMetadata(false);
-    createMetaFile(basePath, HoodieTimeline.makeCommitFileName(instantTime), commitMetadata);
+    createMetaFile(basePath, new DefaultInstantFileNameGenerator().makeCommitFileName(instantTime), commitMetadata);
```

Comment:
It look like we can just make the method `makeCommitFileName` as static.

Reply:
The major consideration for this refactoring was to easily drop 0.x support in later releases. It is not easy to do this piecemeal as dropping this code requires refactoring too. So, intentionally keeping these setup. Thanks